src/sass/components/dsf-header.scss

```diff
@@ -0,0 +1,29 @@
+// -----------------------------------------------------------------------------
+// This file contains all styles related to the header of the site/application.
+// -----------------------------------------------------------------------------
+//$dsf-primary: #31576F;
+$dsf-white: #ffffff;
+@import "../../../src/sass/base/dsf-variables.scss";
+.navbar {
+    background-color: $dsf-white !important;
+   .container-fluid {
```

Comment:
Use a `mixin` to bring in `navbar-expand-lg` `navbar-dark` `bg-dark` into `navbar`/`dsf-navbar`

Reply:
https://sass-lang.com/documentation/at-rules/mixin